src/order/basic.lean

```diff
@@ -29,7 +29,7 @@ classes and allows to transfer order instances.
 
 * `has_sup`: type class for the `⊔` notation
 * `has_inf`: type class for the `⊓` notation
-* `has_compl`: type class for the complement operator
+* `has_compl`: type class for the `ᶜ` notation
```

Comment:
Should `has_sdiff` be in this header too? Or is it defined / described elsewhere?

Reply:
`has_sdiff` is defined in core.